pkg/injector/pod_patch.go

```diff
@@ -142,10 +146,57 @@ func (i *injector) getPodPatchOperations(ar *v1beta1.AdmissionReview,
 			Value: value,
 		},
 	)
+	patchOps = append(patchOps, envPatchOps...)
 
 	return patchOps, nil
 }
 
+func setDaprEnvVars(list []corev1.Container, basePath string) []PatchOperation {
+	envPatchOps := []PatchOperation{}
+	portEnv := []corev1.EnvVar{
+		{
+			Name:  userContainerDaprHTTPPortName,
+			Value: strconv.Itoa(sidecarHTTPPort),
+		},
+		{
+			Name:  userContainerDaprGRPCPortName,
+			Value: strconv.Itoa(sidecarAPIGRPCPort),
+		},
+	}
+	var value interface{}
+	for i, container := range list {
```

Comment:
What happens if pods have multiple containers ? e.g. istio envoy or the other sidecars are already injected ?

Reply:
This code will inject the env vars into all defined/injected containers.